static/dream_web/index.css

```diff
@@ -1,61 +1,274 @@
+    background-color: white;
+    border: solid 2px var(--border_container);
+    border-radius: 20px;
+    box-shadow: 0px 0px 30px -30px black;
+    display: flex;
+    flex-wrap: wrap;
 }
-#search {
-    margin-top: 20vh;
-    margin-left: auto;
-    margin-right: auto;
-    max-width: 800px;
+#search{
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
 
-    text-align: center;
 }
-fieldset {
+#results{
+    width: 60%;
+    align-items: flex-start;
+    justify-content: space-between;
+    flex-direction: row;
+    padding: 20px 30px;
+}
+
+fieldset{
     border: none;
 }
-#fieldset-search {
-    display: flex;
+/*---------------------PROMPT-FIELD---------------------*/
+h2{
+    color: var(--font) !important;
+    padding: 10px 0;
+}
+h2 span{
+    /*background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);*/
+    background: linear-gradient(-450deg, #CC648F, #3C308C, #171640, #6387A6, #A0E3F2);
+	background-size: 200% 200%;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
 }
 #prompt {
+    width: 100%;
+    font-size: 18px !important;
+    border: solid 2px var(--field_border);
+    border-radius: 7px;
+    padding: 15px;
+    color: var(--field_font);
+    font-weight: 600;
+    font-size: 14px;
+    transition: ease all 0.2s;
+}
+#prompt::placeholder{
+    color: var(--placeholder_font);
+    font-size: 16px !important;
+}
+
+#prompt:focus{
+    border: solid 2px var(--field_border_focus);
+    outline: none;
+}
+
+/*---------------------CONFIG FIELDS---------------------*/
+#generate-form {
+    min-width: 100%;
+}
+
+#configs-container{
+    width: 100%;
+    background-color: #f1f1f1;
+    border-radius: 8px;
+    padding: 5px 15px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.config-container{
+    flex-basis: auto;
+    display: flex;
+    flex-direction: column;
     flex-grow: 1;
+    justify-content: space-between;
+    align-items: flex-start;
+    padding: 10px;
+}
+.config-container label{
+    font-size: 14px;
+    font-weight: 500;
+    margin-bottom: 10px;
+}
+
+.config-container input, .config-container select{
+    width: 100%;
+    display: block;
+    text-align: right;
+    border: solid 2px var(--field_border);
+    border-radius: 7px;
+    padding: 5px;
+    color: var(--field_font);
+    font-weight: 600;
+    font-size: 14px;
+    transition: ease all 0.2s;
+}
 
-    border-radius: 20px 0px 0px 20px;
-    padding: 5px 10px 5px 10px;
-    border: 1px solid black;
-    border-right: none;
+.config-container input:focus, .config-container select:focus{
+    border-color: var(--interactivo);
     outline: none;
 }
+
+.config-container input[type=file]{
+    width: 100% !important;
+    flex-basis: auto;
+    padding: 15px;
+    flex-grow: 3;
+    cursor: pointer;
+    background-color: white;
+}
+#seed{
+    width: 110px;
+}
+/*

```

Comment:
Should this commented section remain here?

Reply:
Solved.